robot/Cumulus/resources/ObjectMangerPageObject.py

```diff
@@ -149,4 +149,44 @@ def create_custom_field(self, **kwargs):
 			elif type.lower() == 'formula':
 				self.create_formula_field(kwargs['Field_Name'],kwargs['Formula'])
 			elif type.lower() == 'text':
-				self.create_text_field(kwargs['Field_Name'])
\ No newline at end of file
+				self.create_text_field(kwargs['Field_Name'])
+		
+	
+	@capture_screenshot_on_error
+	def add_picklist_values(self, **kwargs):
```

Comment:
Also, can these picklist values be added as part of org setup?

Reply:
modified it to be part of setup